src/System.Diagnostics.PerformanceCounter/tests/Helpers.cs

```diff
@@ -13,7 +13,7 @@ namespace System.Diagnostics.Tests
     internal class Helpers
     {
         public static bool IsElevatedAndCanWriteToPerfCounters { get => AdminHelpers.IsProcessElevated() && CanWriteToPerfCounters; }
-        public static bool CanWriteToPerfCounters { get => PlatformDetection.IsNotWindowsNanoServer; }
+        public static bool CanWriteToPerfCounters { get => PlatformDetection.IsNotWindowsNanoServer || (PlatformDetection.IsWindows && PlatformDetection.IsArmOrArm64Process); }
```

Comment:
Is this missing a !?  This is saying that we can only write to perf counters if the process is arm or arm64?  Also, do these tests only run on windows?  If so, do we need the IsWindows check?

Reply:
the condition was wrong. This seems to run only on Windows and help is specific to this test set so the IsWindows was not needed.

Reply:
BTW #35743 will stay open so somebody can take closer look.